src/d/actor/d_a_obj_table.cpp

```diff
@@ -4,78 +4,185 @@
 //
 
 #include "d/actor/d_a_obj_table.h"
+#include "d/d_a_obj.h"
+#include "d/d_com_inf_game.h"
 #include "d/d_procname.h"
+#include "f_op/f_op_actor_mng.h"
+
+Mtx daObjTable::Act_c::M_tmp_mtx;
+const char daObjTable::Act_c::M_arcname[6] = "Table";
 
 /* 00000078-000001B8       .text CreateHeap__Q210daObjTable5Act_cFv */
 BOOL daObjTable::Act_c::CreateHeap() {
-    /* Nonmatching */
+    u8 prm;
+    J3DModelData *model_data;
+    J3DModel *model;
+
+
+    prm = daObj::PrmAbstract(this, PRM_LINK_ID_W, PRM_LINK_ID_S);
+    if (!prm) {
+        model_data = (J3DModelData*) dRes_control_c::getRes(M_arcname, 4, g_dComIfG_gameInfo.mResControl.mObjectInfo, 0x40);
+        
+        JUT_ASSERT(0x51, model_data != 0);
+        
+        model = mDoExt_J3DModel__create(model_data, 0, 0x11020203);
+        mpModel = model;
+    } else {
+        model_data = (J3DModelData *) dRes_control_c::getRes(M_arcname, 5, g_dComIfG_gameInfo.mResControl.mObjectInfo, 0x40);
+        
+        JUT_ASSERT(0x57, model_data != 0);
+        
+        model = mDoExt_J3DModel__create(model_data, 0, 0x11020203);
+        mpModel = model;
+    }
+    return this->mpModel != 0;
 }
 
 /* 000001B8-00000284       .text Create__Q210daObjTable5Act_cFv */
 BOOL daObjTable::Act_c::Create() {
-    /* Nonmatching */
+    u8 prm;
+    dBgW *coll;
+    uint coll_id;
+    u8 release;
+
+    cullMtx = mpModel->mBaseTransformMtx;
+    init_mtx();
+
+    fopAcM_setCullSizeBox(this, -80.0f, -1.0f, -80.0f, 80.0f, 205.0f, 80.0f);
+    prm = daObj::PrmAbstract(this, PRM_LINK_ID_W, PRM_LINK_ID_S);
+
+    if (((uint)prm == '\x02') && (coll = mpBgW, coll != NULL)) {
```

Comment:
The mdl param here doesn't make sense as an ascii character '\x02', that's a Ghidra-ism. You should write it as the integer 2.
```suggestion
    if (prm_get_mdl() == 2 && (coll = mpBgW, coll != NULL)) {
```

Also, we don't want assignments inside of if statements, that's another Ghidra-ism.
```suggestion
    if (prm_get_mdl() == 2) {
        coll = mpBgW;
        if (coll != NULL) {
```

Though I doubt this temp variable is necessary, so you could probably just write:
```suggestion
    if (prm_get_mdl() == 2 && mpBgW != NULL) {
```